app/models/expense.rb

```diff
@@ -2,6 +2,12 @@
 
 # Expense Model
 class Expense < ApplicationRecord
+  include SqlHelper
+
   belongs_to :budget_plan
   belongs_to :category
+
+  def sanitize_sql_query(sql, *params)
```

Comment:
instead of exposing `sanitize_sql_query`, what about creating a class method that returns the `Expense` by plan `id`?

Reply:
this isnt used at the moment but its here in case i have other future raw sql i want to run